importer/importer.py

```diff
@@ -260,6 +265,13 @@ def get_items(connection,
         print("\n")  # linebreak needed in case of visual feedback dots
     if problem_reports:
         print("SAVED PROBLEM REPORTS TO {}".format(filenames['reports']))
+    if skipped_uploads:
+        skipped_items_output += "\n".join(
```

Comment:
`skipped_items_output` is never initialized so it will crash because referenced before assignment.